### PR TITLE
Adhere to latest go-ipfs API; use gx exclusively

### PR DESCRIPTION
**Pull request by hackergrrl**
Hey @whyrusleeping o/

I finally have this repo fully under gx for dependencies, and updated the go-ipfs API usage to match the changes over the last several months. This will make `ipget` very happy.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "author": "noffle",
+  "bugs": {},
+  "gx": {
+    "dvcsimport": "github.com/noffle/ipfs-embedded-shell"
```
Comment (whyrusleeping) [inline package.json]: probably want to change the path here to point to my repo (otherwise deps rewriting won't work properly)

**Comment by hackergrrl** [comment]
Fixed. Good catch.

**Comment by whyrusleeping** [comment]
Cool, thanks! Sorry this took so long to merge

**Comment by hackergrrl** [comment]
Thanks Jeromy!

**Comment by hackergrrl** [comment]
What do you think about granting me push perms to this repo? It would help me iterate more quickly on ipget fixes/updates.

**Comment by whyrusleeping** [comment]
Yeah, sure thing. Just added you as a collaborator. I also pushed a commit to this with updated dependencies

**Comment by hackergrrl** [comment]
Thanks!

**Pull request merged**